dataobjects/layers.py

```diff
@@ -46,4 +49,13 @@ def create(self):
         return self.__layer
 
     def table_name(self):
-        return QgsDataSourceUri(self.uri).table()
\ No newline at end of file
+        return QgsDataSourceUri(self.uri).table()
+
+    def inner_id(self):
```

Comment:
Can you add some small documentation that explains what this is used for (i.e. only within code here and yaml -> or also qgis project file).
I know I haven't been very good at documenting code but if more people start working on this, we really will need to improve on this kind of quality (myself included).